src/Engines/MeiliSearchEngine.php

```diff
@@ -54,6 +63,10 @@ public function update($models)
 
         $index = $this->meilisearch->index($models->first()->searchableAs());
 
+        if ($settings = $this->settings[$models->first()->searchableAs()] ?? []) {
+            $index->updateSettings($settings);
+        }
+
```

Comment:
I don't think this would be super beneficial to users. Yes, this would keep index settings up to date. BUT everytime you update a model, this will add a update task to the meilisearch queue too (if there were no changes to the settings, there are would be no consequences and no performance degradation for meili users).

Still every model update will create the following tasks in meilisearch:

- update index settings
- update model/s

In any case, I would recommend everyone that deploys meilisearch to production with Laravel Scout to create a custom command that handles updates to the index settings (just like any migration command for the database).

Reply:
Thanks @mmachatschek. You highlighted the one thing I was concerned about. I'll think about this a bit more and perhaps introduce a new command to update an index.

Reply:
I've added a new `scout:sync-settings` command that can be used during deployment. Can you review again?

Reply:
@driesvints this is a nice addition. thats basically the same code that I use in my environment. I didn't test your code but it looks rock solid 👍